<launch>
    <node pkg="sinwave" name="generator" type="generator.py" output="screen">
        <param name='outTopic' type='string' value='sine'/>
        <param name='frequency' type='double' value='1'/>
        <param name='executionFrequency' type='double' value='30.0'/>
    </node>

    <node pkg="sinwave" name="logger" type="logger.py" output="screen">
        <param name='inTopic' type='string' value='sine'/>
    </node>

    <node name="loggerRqt" pkg="rqt_plot" type="rqt_plot" args="/sine/value"/>
</launch>
